examples/connectivity/plot_adhd_covariance.py

```diff
@@ -26,7 +26,7 @@ def plot_matrices(cov, prec, title):
 
     # Put zeros on the diagonal, for graph clarity.
     size = prec.shape[0]
-    prec[range(size), range(size)] = 0
+    prec[list(range(size)), list(range(size))] = 0
```

Comment:
How about opening an issue to track line changes for when the version gets bumped?  I'd be happy to do that, if that sounds useful.